ecosystem/indexer-grpc/indexer-grpc-table-info/src/runtime.rs

```diff
@@ -13,6 +13,26 @@ use tokio::runtime::Runtime;
 
 const INDEX_ASYNC_V2_DB_NAME: &str = "index_indexer_async_v2_db";
 
+pub fn bootstrap_db_tailer(
+    config: &NodeConfig,
+    db_rw: DbReaderWriter,
+) -> Option<(Runtime, Arc<DBTailer>)> {
+    if !config.indexer_table_info.enabled {
```

Comment:
We might need a separate flag for the rollout / migration? Otherwise after a binary rollout a node will suddenly lose all the index data?